### DISCUSSION TITLE
Finding fused transcripts using UCSCXenaTools

### DISCUSSION THREAD

**Discussion by Volkan-Ergin** [ShixiangWang/self-study#94] [Q&A] [answered]
Hi Shixiang, I came across your web pages describing the UCSCXenaTools. I am wondering how I can utilize the package in R to retrieve/detect fused transcripts from the Treehouse dataset. Could you provide an example illustrating how to use UCSCXenaTools to study transcript fusions? If it's unable to do so, do you think I can use STAR-Fusion or something similar after downloading the Treehouse data through UCSCXenaTools? Since GEO provides only expression matrix and metadata for the Treehouse (GSE129326), I guess I need to start with the XenaTools to import all the necessary data into R, right?
Thank you.

**Comment by ShixiangWang** [comment, marked as answer]
@VolSapiens Hi, the UCSCXenaTools serves as a downloader for UCSC Xena datasets and loader for preparing the downloaded datasets for R.  

You can check [this page](https://xenabrowser.net/datapages/?host=https%3A%2F%2Fxena.treehouse.gi.ucsc.edu%3A443&removeHub=https%3A%2F%2Fxena.treehouse.gi.ucsc.edu%3A443) for the treehouse cohort datasets available in UCSC Xena. I cannot find any dataset related to fused transcript. So this tool is useless for you. ([treehouse home](https://treehousegenomics.ucsc.edu/))

My suggestion is that you find and download interested raw data (FASTQ and BAM) from treehouse (you should search the raw data link in original paper or treehouse portal), and then use software like STAR-fusion to find fused transcript.

Another way is search papers that have already processed fused transcripts from public datasets including treehouse (e.g., https://tumorfusions.org/).

If the raw data is not available. You should contact the treehouse team.

Best,

Shixiang

**Reply by Volkan-Ergin** [reply]
Thank you Shixiang for your reply.
